projects/packages/jetpack-mu-wpcom/bin/sync-newspack-blocks.sh

```diff
@@ -61,7 +61,10 @@ then
 
 		if [[ "$CURRENT_VERSION" == "$NAME" ]]; then
 			echo "The current version $CURRENT_VERSION of the newspack-blocks is synced."
-			exit 0
+			read -rp "Do you want to proceed anyway? (y/N): " proceed
```

Comment:
If you want you could source the file from the monorepo root `tools/includes/proceed_p.sh` to use our fancy function for doing prompts like this.

Reply:
Any advantage of sourcing vs a simple `read`? :^)

Reply:
The `proceed_p` function handles a bunch of edge cases. It doesn't matter a whole lot though. 🤷